podcasts/CreateFolderView.swift

```diff
@@ -64,6 +64,14 @@ struct CreateFolderView: View {
                         }
                         .accessibilityLabel(L10n.close)
                     }
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button {
+                            pickerModel.toggleSelectAll()
+                        } label: {
+                            Text(pickerModel.hasSelectedAll ? L10n.deselectAll : L10n.selectAll)
+                        }
+                        .foregroundColor(ThemeColor.secondaryIcon01(for: theme.activeTheme).color)
```

Comment:
Not related to this PR but just leaving it here for future reference: I believe how we can refer to theme colors can be better.

The way we do now we:

1. Refer to another static class `ThemeColor`
2. Pass the active theme as an argument
3. Make the conversion to `Color`

It would be better if we could just do `theme.secondaryIcon01` or something like that.